platform/web/detect.py

```diff
@@ -77,26 +77,18 @@ def configure(env):
         sys.exit(255)
 
     ## Build type
-    if env["target"].startswith("release"):
-        if env["optimize"] == "size":
-            env.Append(CCFLAGS=["-Os"])
-            env.Append(LINKFLAGS=["-Os"])
-        elif env["optimize"] == "speed":
-            env.Append(CCFLAGS=["-O3"])
-            env.Append(LINKFLAGS=["-O3"])
-
-        if env["target"] == "release_debug":
-            # Retain function names for backtraces at the cost of file size.
-            env.Append(LINKFLAGS=["--profiling-funcs"])
-    else:  # "debug"
-        env.Append(CCFLAGS=["-O1", "-g"])
-        env.Append(LINKFLAGS=["-O1", "-g"])
+
+    # TODO: Re-evaluate the need for this / streamline with common config.
+    if not env["target"].endswith("-dev"):
+        # Retain function names for backtraces at the cost of file size.
+        env.Append(LINKFLAGS=["--profiling-funcs"])
```

Comment:
Noticed an unintended logic change here, this used to be added only for `release_debug` so it should check `template-debug` or `editor`.

Reply:
Fixed.